uspace/lib/inet/meson.build

```diff
@@ -49,3 +49,4 @@ test_src = files(
 	'test/eth_addr.c',
 	'test/main.c',
 )
+allow_shared = true
```

Comment:
This is a complete nitpick, but I feel like there should be an empty line after the closing parenthesis.